js/app.js

```diff
@@ -2,14 +2,15 @@
 /**
  * Construct and fill Product Array
  */
-var urls = ['../img/bag.jpg', '../img/banana.jpg', '../img/bathroom.jpg', '../img/boots.jpg', '../img/breakfast.jpg', '../img/bubblegum.jpg', '../img/chair.jpg', '../img/cthulhu.jpg', '../img/dog-duck.jpg', '../img/dragon.jpg', '../img/pen.jpg', '../img/pet-sweep.jpg', '../img/scissors.jpg', '../img/shark.jpg', '../img/sweep.png', '../img/tauntaun.jpg', '../img/unicorn.jpg', '../img/usb.gif', '../img/water-can.jpg', '../img/wine-glass.jpg'];
+var urls = ['img/bag.jpg', 'img/banana.jpg', 'img/bathroom.jpg', 'img/boots.jpg', 'img/breakfast.jpg', 'img/bubblegum.jpg', 'img/chair.jpg', 'img/cthulhu.jpg', 'img/dog-duck.jpg', 'img/dragon.jpg', 'img/pen.jpg', 'img/pet-sweep.jpg', 'img/scissors.jpg', 'img/shark.jpg', 'img/sweep.png', 'img/tauntaun.jpg', 'img/unicorn.jpg', 'img/usb.gif', 'img/water-can.jpg', 'img/wine-glass.jpg'];
```

Comment:
In the future, if you realize you're working on the wrong branch, you can always checkout a new branch right then!